src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let numbers = [];
+
+  const adder = number => {
+    if (number !== undefined) {
+      numbers.push(number);
+
+      return adder;
+    }
+
+    const result = numbers.reduce((a, b) => (a + b), 0);
```

Comment:
will be better for the algorithm, if you just create counter & just sum these numbers